helpers/lib/fonts.js

```diff
@@ -110,8 +112,10 @@ module.exports = function(format, themeSettings, handlebars) {
     // Format output based on requested format
     switch(format) {
     case 'linkElements':
+        const options = arguments.length > 3 ? arguments[arguments.length - 1] : null;
+        const fontDisplay = options && options.hash['font-display'] ? options.hash['font-display'] : null;
```

Comment:
Please move this use of `arguments` up to the helper level (in `getFontsCollection`). Since this is library code and not an external interface with theme code, we have more flexibility in changing the interface and don't need to rely on this hack. Instead, it would be better to add an `options` object as an optional fourth parameter to this function, and then create these options in `getFontCollection` as needed.

Reply:
Removed arguments line.

Reply:
Thanks for the suggestions, I can't believe I didn't think of the first one lol.